### ISSUE TITLE
OSC Commands not working

### ISSUE THREAD

**Issue by thespica93** [xLightsSequencer/xLights#1706]
**Describe the bug**
I am trying to fire OSC Commands from my iPhone and iPad to trigger specific playlist. I started small with 1 playlist and 3 different sequences. I created the TouchOSC app and transfered it to my phone. I have mapped the path of each button to XSchedule as an event. I have configured OSC in xSchedule.  Nothing is working. 

**To Reproduce**

I have tried to do this on 2 different laptops with 3 separate wireless routers. 

**Expected behavior**
When I press the button on my iphone, it should trigger one of my sequences. 

**Screenshots**
![Xschedule Picture](https://user-images.githubusercontent.com/42631192/64730627-c02b3a80-d4ad-11e9-8b8f-62f701490f46.png)
![Configure OSC Picture](https://user-images.githubusercontent.com/42631192/64730670-cae5cf80-d4ad-11e9-8248-982c3050f438.png)
![Events Picture](https://user-images.githubusercontent.com/42631192/64730709-d0431a00-d4ad-11e9-8b4d-2a618c1b949d.png)
![Events Config Picture](https://user-images.githubusercontent.com/42631192/64730735-d33e0a80-d4ad-11e9-8b65-14e29429e445.png)
![TouchOSC picture](https://user-images.githubusercontent.com/42631192/64730793-dd600900-d4ad-11e9-967e-303ec665ba59.png)
![Cell OSC Config](https://user-images.githubusercontent.com/42631192/64730959-3e87dc80-d4ae-11e9-814e-13f96adc653c.png)

**Versions (please complete the following information):**
 - OS: Windows 10
 - xLights version 2019.46

THANK YOU FOR ALL OF YOUR WORK!

**Comment by alentz44** [comment]
This is way late but after fighting with this for a while, I found that the path has to be at least 10 characters long to work. When I used less than 10 characters in the path I received the following xscheduler logged error: 
`9192 log_base [DEBUG] 2f 50 75 72 70 6c 65 70 31 00 00 00 2c 00 00 00     /Purple...,...`

When using 10 or more characters I received success. Note the number of objects or types of characters in the path didn't seem to make a difference (only the number of characters). Logged Success: 
`2023-07-24 16:04:42,660 9192 log_base [DEBUG] OSC Path: /Purple123.`

Versions used:
touch OSC: 1.2.2.175
XLights (I love this product! thanks Keith and all those that support it): 2023.11
